search/depth_first_search.py

```diff
@@ -11,11 +11,12 @@
 }
 
 
+
 def dfs(graph, node):
-    visited = []
+    visited = set()
```

Comment:
can you add a comment above this saying `# the video has visited as an array. I changed this to a set because....`